jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

```diff
@@ -284,7 +295,7 @@ pipeline {
                 catchError(stageResult: 'FAILURE') {
                     archiveArtifacts allowEmptyArchive: true, artifacts: 'log/*report.xml, log/*report.html, support_bundle/*.tar, crash_files/*.gz, CHANGESET.txt', followSymlinks: false
                     emailext (
-                        body: '''${SCRIPT, template="K8s-deployment-email_3.template"}${SCRIPT, template="REL_QA_SANITY_CUS_EMAIL_7.template"}''',
+                        body: '''${SCRIPT, template="K8s-deployment-email_4.template"}${SCRIPT, template="REL_QA_SANITY_CUS_EMAIL_7.template"}''',
```

Comment:
In Deployment info section: CORTX IMAGEs can be labeled as -> CORTX Images:  to make it consistent

Reply:
Addressed review comment -  https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/sv_space/job/nightly-ci-cd/344/console 

![image](https://user-images.githubusercontent.com/63289460/179747670-1ddc8f65-23d9-4117-8bab-47938207dbf5.png)